### PR TITLE
Upgrade to storybook v5

### PR DESCRIPTION
**Pull request by emmenko**
Storybook v5 is out! (https://medium.com/storybookjs/storybook-5-0-db1d0f9c83b8)

screenshot 2019-03-06 at 22 11 29

#### Storybook Readme

The readme addon doesn't support v5 yet, but there is an open issue to work on that: https://github.com/tuchk4/storybook-readme/issues/118

I'll keep this PR as a draft until there is a release from that package.

### CODE REVIEW

**Comment by tdeekens** [comment]
Yeah, cool. It looks loads better. When I updated to the beta a couple of weeks I only noticed that the panel arrangement changed and didnt seem configurable. Nothing to hold back the update obviously but if I recall correctly the knobs panel can not be right any more.

**Comment by emmenko** [comment]
It's the same as before actually, you can toggle the panel on/off (check out the deploy preview).

**Comment by emmenko** [comment]
Storybook readme got released with support for v5. I think we are good to go now 🚀

File: package.json [hunk 1]
```diff
@@ -96,13 +96,14 @@
...
     "@emotion/babel-preset-css-prop": "10.0.7",
     "@percy/puppeteer": "1.0.5",
-    "@storybook/addon-actions": "4.1.13",
-    "@storybook/addon-info": "4.1.13",
-    "@storybook/addon-knobs": "4.1.13",
-    "@storybook/addon-links": "4.1.13",
-    "@storybook/addon-options": "4.1.13",
-    "@storybook/addon-storysource": "4.1.13",
-    "@storybook/react": "4.1.13",
+    "@storybook/addon-actions": "5.1.0-alpha.8",
```
Comment (tdeekens) [inline package.json]: We should move to stable from alpha before merging, correct?

**Comment by tdeekens** [comment]
⚠️ @emmenko branch got rebased and we now use latest stable storybook@5. Will merge after tests pass ⚠️

**Comment by adnasa** [comment]
> Will merge after tests pass

Shall we go ahead with this? my impression was that we wait for @montezume. 
But sure let's go for it

**Pull request merged**